### PR TITLE
AB-2320633-SubmittedAt property to be included in LTI Testing Tool

### PR DESCRIPTION
**Pull request by skarunanithi20**
Reference ticket - https://dev.azure.com/AnthologyInc-01/Learn/_workitems/edit/2123831 
Including submittedAt property in LTI Testing Tool to help testing & development :

image

### CODE REVIEW

File: server/src/app/assign-grades.js [hunk 1]
```diff
@@ -283,7 +284,8 @@ export const scores = (req, res, agPayload, task) => {
           comment: 'This is exceptional work.',
           timestamp: '2017-04-16T18:54:36.736+00:00',
           activityProgress: activityProgress,
-          gradingProgress: gradingProgress
+          gradingProgress: gradingProgress,
+          submittedAt: submittedAt
```
[the remark this answers is not in the document]
Comment (mark-b-kauffman) [reply server/src/app/assign-grades.js:288]: Ditto Varju's comment.

File: server/src/app/assign-grades.js [hunk 2]
```diff
@@ -283,7 +284,8 @@ export const scores = (req, res, agPayload, task) => {
           comment: 'This is exceptional work.',
           timestamp: '2017-04-16T18:54:36.736+00:00',
```
[the remark this answers is not in the document]
Comment (mark-b-kauffman) [reply server/src/app/assign-grades.js:285]: Ditto Varju's comment.

File: server/src/app/assign-grades.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (skarunanithi20) [reply server/src/app/assign-grades.js:288]: It seems like there might be reason to want to test handling of a missing value on the platform side : Yes - If submittedAt is missing then timestamp is substituted in place of submittedAt value in learn backend. 

Noted @varju . Will add submittedAt as optional field.

File: server/src/app/assign-grades.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (skarunanithi20) [reply server/src/app/assign-grades.js:285]: Will change to the required format. Thanks for noting.

**Review by varju — APPROVED** [review]

**Review by obedlivi24 — COMMENTED** [review]
approved

**Review by abasith-bbdn — COMMENTED** [review]
approved

**Pull request merged**